scripts/build-webpack.js

```diff
@@ -33,7 +36,7 @@ const compiler = webpack({
     }),
   ],
   output: {
-    filename: `yarn-${version}.js`,
+    filename: `[name]-${version}.js`,
```

Comment:
> because I think that is the clearest and easiest system to understand.

I do the same thing for one of my C# projects - Every package uses the same version number, even if it doesn't change as part of a release. Makes it easier to reason about the version number.

> Deciding on the version number of the lockfile package and whether or not to publish a new version becomes one more manual task that could be forgotten instead of part of the regular release process.

On the other hand, we might want to have a proper schema for the lockfile, and bump its major version separately for any major lockfile changes. Hmmmmmm

I'll leave this up to the other Yarn team members; I'm happy either way 😃